include/deal.II/base/symmetric_tensor.h

```diff
@@ -709,7 +734,7 @@ class SymmetricTensor
   /**
    * Return an element by value.
    */
-  Number operator() (const TableIndices<rank_> &indices) const;
+  const Number &operator() (const TableIndices<rank_> &indices) const;
```

Comment:
This no longer corresponds to the documentation of the function. Maybe rewrite this: "Return a @p const reference to the value referred to by the argument."?